rust/lance-encoding/src/buffer.rs

```diff
@@ -298,13 +204,16 @@ impl LanceBuffer {
             data.extend_from_slice(buffer.as_ref());
         }
 
-        Self::Owned(data)
+        Self(Buffer::from_vec(data))
     }
 
     /// Zips multiple buffers into a single buffer, consuming the input buffers
     ///
     /// Unlike concat_into_one this "zips" the buffers, interleaving the values
     pub fn zip_into_one(buffers: Vec<(Self, u64)>, num_values: u64) -> Result<Self> {
+        use lance_core::Error;
+        use snafu::location;
+
```

Comment:
Can we just keep these `use` statements at the top?  This isn't in a `cfg` block so I'm not sure we're saving much.